lib/rototiller/utilities/command_flag.rb

```diff
@@ -32,4 +34,21 @@ def describe_flag_state
     flag_with_value = "The CLI flag #{@flag} will be used with value #{@value}."
     green_text(@value.nil? ? only_flag : flag_with_value)
   end
+
+  def validate_attribute_hash(h)
+    # validate the contents of the hash
+    name_error = 'A name must be supplied to add_flag'
+    value_error = 'A value must be supplied to add flag'
+    error = nil
+
+    if !h[:name] && !h[:value]
+      error = name_error << "\n" << value_error
+    elsif h[:name] && !h[:value]
+      error = value_error
+    elsif !h[:name] && h[:value]
+      error = name_error
+    end
+
+    raise(ArgumentError, error) if error
+  end
```

Comment:
@er0ck I agree.  The values are in flux at this point :value ~ :default.  Should this work be ticketed?